### PR TITLE
Contacts

### PR DESCRIPTION
**Pull request by bananateg**
At this moment is posiible add more the one contact, i am using a library https://github.com/broakenmedia/MultiContactPicker, I have 2 bugs, the resultCode of use by taken the result and the color of new screen

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -128,6 +128,8 @@ dependencies {
 
     // (Java only)
     implementation "androidx.work:work-runtime:$work_version"
+
```
Comment (aakash1313) [inline app/build.gradle:131]: @bananateg : Please don't use third party libraries, they can cause issues later or not. Unless we can't achieve the same through native , please refrain from using them.

Comment (justinklchan) [reply app/build.gradle:131]: I agree with @aakash1313 let's find a way to use native libraries.
Using a third party library for our calendar has caused many crashes in production.

Comment (bananateg) [reply app/build.gradle:131]: ok

**Comment by justinklchan** [comment]
i'm going to merge this for now, the issue with our calendar is not related to it being a third party library.

if we are able to have a better way with native, that is great. but for now let's use third party. and if we find it doesn't perform well during testing we can remove it.

**Pull request merged**